src/test/fixtures/profiles/zip-file.js

```diff
@@ -3,24 +3,31 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 // @flow
 
-import { getEmptyProfile } from '../../../profile-logic/profile-data';
+import { getProfileFromTextSamples } from '../../fixtures/profiles/make-profile';
 import { serializeProfile } from '../../../profile-logic/process-profile';
 import { receiveZipFile } from '../../../actions/receive-profile';
 import type { ZipFileTable } from '../../../profile-logic/zip-files';
 import createStore from '../../../create-store';
 import JSZip from 'jszip';
-
+import { objectValues } from '../../../utils/flow';
 /**
  * Puts a blank profile at each given path in a zip file.
  */
 export function getZippedProfiles(files: string[] = []): JSZip {
-  const profile = serializeProfile(getEmptyProfile());
+  const { profile } = getProfileFromTextSamples('A');
+  const profileText = serializeProfile(profile);
 
   const zip = new JSZip();
   files.forEach(fileName => {
-    zip.file(fileName, profile);
+    zip.file(fileName, profileText);
   });
 
+  for (const file of objectValues(zip.files)) {
+    // The date for files, and directories defaults to the current time,
+    // which breaks snapshot tests.
+    file.date = new Date('1997-08-29T05:14:14.617Z');
```

Comment:
It's probably better to set the date directly in the call to `zip.file` above:
```javascript
zip.file(fileName, profileText, { date: new Date('1997-08-29T05:14:14.617Z') });
```
(see https://stuk.github.io/jszip/documentation/api_jszip/file_data.html#date-option)

Reply:
I did this originally, but it doesn't set the directories, and the tests fail.